lib/integration/prismaConnector.ts

```diff
@@ -22,7 +22,7 @@ const prismaClientSingleton = () => {
       template: {
         $allOperations({ args, query }) {
           if ("where" in args) {
-            args.where = { ...args.where, ttl: null };
+            args.where = { ...args.where };
```

Comment:
@thiessenp-cds 

Looks like you have the most context on the `ttl` here ...

https://github.com/cds-snc/platform-forms-client/pull/5339/files

Concerns with removing?